### ISSUE TITLE
Domain discriminator

### ISSUE THREAD

**Issue by QHxingchen** [bennyguo/sketch2model#3]
Thanks for your work, I have a question about your implementation of the domain discriminator, I use the two sketch datasets you recommend in your paper, and use the domain discriminator recommended in your project, use 512×7×7 feature maps for channel attention, as done in CBAM, flattened as input to the domain discriminato, but I still can't achieve the volXel IoU in your paper after adding the domain discriminator on the airplane class, my way only achieves 0.490, 0.486. Can you give me some suggestions? Looking forward to your reply!

**Comment by bennyguo** [comment]
Hi, sorry for the late reply!

We do find training directly with the domain discriminator to be somehow unstable. Please try training the model without the domain discriminator first then finetune with it under a low learning rate.
